Sourcery/Models/Type.swift

```diff
@@ -245,6 +251,8 @@ class Type: NSObject, AutoDiffable, Annotated, NSCoding {
 }
 
 extension Type {
+
+    // sourcery: skipDescription
```

Comment:
Don't think if this property is used anywhere, so maybe we don't need it at all.